src/Kushy/SymbolLoader.cs

```diff
@@ -42,11 +42,41 @@ public async Task<string[]> GetDatabaseNamesAsync(string clusterName = null, boo
             var connection = GetClusterConnection(clusterName);
             var databases = await ExecuteControlCommandAsync<ShowDatabasesResult>(connection, "", ".show databases", throwOnError, cancellationToken);
             if (databases == null)
-                return null;
+            {
+                throw new Exception("No databases found for cluster or No access to the cluster: " + clusterName);
+            }
 
             return databases.Select(d => d.DatabaseName).ToArray();
         }
 
+        /// <summary>
+        /// checks for equivalent named clusters in current loaded globals
+        /// </summary>
+        public ClusterSymbol GetClusterSymbolFromGlobals(string clusterName, GlobalState globals)
```

Comment:
If you are trying to add the ability to look up the cluster by its short name (without .Kusto.Windows.Net suffix) then this is not how to do it.

Reply:
This is no longer needed. removed.